docs/src/main/paradox/release-notes.md

```diff
@@ -1,5 +1,28 @@
 # Release Notes
 
+## 10.0.8
+
+### Ability to express textual content types with missing character set
+
+Akka-http has a strongly typed media type / content type system, and knows at compile time about which media types
+are supposed to express a character set attribute, e.g. `text/plain; charset=UTF-8`. Before this release, akka would
+silently assume UTF-8 for `ContentType` instances of media types with a missing `charset` attribute.
+
+From now on, content types missing a charset can be both parsed and expressed directly, using the new 
+`ContentType.WithMissingCharset` trait/class. 
+
+- If you happen to parse an http request with just `text/plain` as its `Content-Type` header, it'll 
+  show up as an instance of this class.
```

Comment:
A bit too casual for release notes, how about:

> For incoming Content-Type headers with values missing a charset, such as `text/plain` the header 
`ContentType` will be represented as `WithMissingCharset`, rather than like previously as a content typed with an assumed UTF-8 charset (which could have been a wrong guess.